android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1342,11 +1342,13 @@ public void onPageFinished(WebView view, String url) {
           public void run() {
             SharedPreferences prefs = context.getSharedPreferences(context.getString(R.string.kma_prefs_name), Context.MODE_PRIVATE);
             if (prefs.getBoolean(KMManager.KMKey_ShouldShowHelpBubble, true)) {
-              InAppKeyboard.loadUrl("javascript:showHelpBubble()");
+              InAppKeyboard.loadJavascript("showHelpBubble()");
             }
           }
         }, 2000);
 
+        InAppKeyboard.callJavascriptAfterLoad();
```

Comment:
No `SystemKeyboard` version yet?

Reply:
Nup, just testing inapp to start with

Reply:
Ah, actually, my bad. The rest should be in place already.

Reply:
Fixed